src/main/java/org/opencds/cqf/tooling/terminology/CodeSystemLookupDictionary.java

```diff
@@ -122,7 +124,7 @@ public static String getUrlFromOid(String oid) {
             case "2.16.840.1.113883.6.21": return "http://terminology.hl7.org/CodeSystem/nubc-UB92";
             case "2.16.840.1.113883.6.301.3": return "http://terminology.hl7.org/CodeSystem/v2-0456";
             case "2.16.840.1.113883.6.50": return "http://terminology.hl7.org/CodeSystem/POS";
-            case "2.16.840.1.113883.6.238": return "http://terminology.hl7.org/CodeSystem/PHRaceAndEthnicityCDC";
+            case "2.16.840.1.113883.6.238": return "https://terminology.hl7.org/3.1.0/NamingSystem-CDCREC.html";
```

Comment:
Per THO link: https://terminology.hl7.org/CodeSystem-CDCREC.html
The URL here should be: urn:oid:2.16.840.1.113883.6.238